### PR TITLE
docs: Update api documentation

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches
- [x] Docs

**What changes will this PR take into?**

Update api documentation

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by juzhiyuan** [comment]
😳

File: docs/en/latest/api/api.md [hunk 1]
```diff
@@ -21,351 +21,881 @@ title: API doc of Manager API.
...
+## Description
+
 Manager API directly operates ETCD and provides data management for Apache APISIX, provides APIs for Front-end or other clients.
 
-**License:** [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+We could pass parameters to APIs to control APISIX Nodes. To have a better understanding about how it works, please see [README](https://github.com/apache/apisix-dashboard/blob/master/README.md).
 
-### /apisix/admin/check_ssl_cert
+9000 is the default port on which the Manager API listens. You could take another port by modifying the [conf/conf.yaml](https://github.com/apache/apisix-dashboard/blob/master/api/conf/conf.yaml) file.
+
```
Comment (starsz) [inline docs/en/latest/api/api.md:31]: Need a table of contents

File: docs/en/latest/api/api.md [hunk 2]
```diff
@@ -21,351 +21,881 @@ title: API doc of Manager API.
...
 ##### Parameters
 
 | Name | Located in | Description | Required | Schema |
-| ---- | ---------- | ----------- | -------- | ------ |
-| cert | body       | cert of SSL | Yes      | string |
-| key  | body       | key of SSL  | Yes      | string |
+| ---- | ---------- | ----------- | -------- | ---- |
+| page | query | The current page number  | No | string |
+| page_size | query | The number of data returned per page | No | string |
+| hostname | query |  | No | string |
```
Comment (starsz) [inline docs/en/latest/api/api.md:103]: Need a description of the `hostname` field.

File: docs/en/latest/api/api.md [hunk 3]
```diff
@@ -21,351 +21,881 @@ title: API doc of Manager API.
...
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| id | path | The id of route | Yes | string |
+
+##### Responses
 
-##### Summary:
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 200 | success | [route](#route) |
```
Comment (starsz) [inline docs/en/latest/api/api.md:303]: The response is `route` looks not very good.

File: docs/en/latest/api/api.md [hunk 4]
```diff
@@ -21,351 +21,881 @@ title: API doc of Manager API.
...
 
-| Name      | Located in | Description    | Required | Schema  |
-| --------- | ---------- | -------------- | -------- | ------- |
-| page      | query      | page number    | No       | integer |
-| page_size | query      | page size      | No       | integer |
-| name      | query      | name of route  | No       | string  |
-| uri       | query      | uri of route   | No       | string  |
-| label     | query      | label of route | No       | string  |
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
```
Comment (starsz) [inline docs/en/latest/api/api.md:490]: Miss example.

**Review by starsz — COMMENTED** [review]
It seems that miss some API such as `/apisix/admin/services`

**Comment by nic-chen** [comment]
> It seems that miss some API such as `/apisix/admin/services`

There is too much content. This pr only contains the differences from the Admin API and the content of the route.

File: docs/en/latest/api/api.md [hunk 5]
```diff
@@ -21,351 +21,881 @@ title: API doc of Manager API.
 #
 -->
 
+## Description
```
Comment (gxthrj) [inline docs/en/latest/api/api.md:24]: This document is too long. 
If there is an increase in API in the future, this article will become longer. 

I hope that the article will be divided into different articles according to the management object.

**Pull request closed without merging**